source/Ship.cpp

```diff
@@ -3550,10 +3554,12 @@ int Ship::TakeDamage(vector<Visual> &visuals, const DamageDealt &damage, const G
 	else if(heat < .9 * MaximumHeat())
 		isOverheated = false;
 
+	double ionWeaponJamChance = damage.Ion() ? CalculateJamChance(Energy(), ionization) : 0.;
 	// If this ship did not consider itself an enemy of the ship that hit it,
 	// it is now "provoked" against that government.
 	if(sourceGovernment && !sourceGovernment->IsEnemy(government)
-			&& (Shields() < .9 || Hull() < .9 || !personality.IsForbearing())
+			&& (!personality.IsForbearing() || Shields() < .9 || Hull() < .9
+				|| isOverheated || ionWeaponJamChance > 0.1 || (slowness > 10. && damage.Slowing()))
```

Comment:
Why only if you are dealing slowing damage?
Surely, taking advantage of the ship being slowed by a hazard to attack it should be enough on its own, regardless of if you are adding slowing yourself.

Reply:
but not if its being ionized?